src/containers/pages/FocusInvestigation/active/style.css

```diff
@@ -2,3 +2,7 @@ div.routine-heading div.row button.focus-investigation.btn.btn-primary {
   margin-top: 3rem !important;
   float: right;
 }
+.ReactTable .-pagination {
```

Comment:
Yaas, do we have a global stylesheet? Could go there

Reply:
Maybe right in here https://github.com/onaio/reveal-frontend/blob/master/src/App/App.css

Reply:
Could we instead have a tables css file?

Reply:
Suuuure.

Reply:
@moshthepitt I moved it into `src/App/tables.css` which is imported into `src/App/App.css`, how does that sound?

Reply:
Or are you thinking of having a `tables.css` which gets imported into pages/components that need it (rather than being global)?

Reply:
> Or are you thinking of having a tables.css which gets imported into pages/components that need it (rather than being global)?

I am leaning towards this.  How do you feel about this?

Reply:
I'm fine with it, there's a bit of a trade-off and it's probably the best thing to do :)